data/schemas/keyboard.jsonschema

```diff
@@ -90,6 +90,7 @@
                 "STM32L442",
                 "STM32L443",
                 "GD32VF103",
+                "FS026",
```

Comment:
Remove this change. There's only one board with this at the moment, and we'll determine if/when we need to add an explicit entry. Use https://github.com/qmk/qmk_firmware/blob/master/keyboards/kinesis/kint41/rules.mk for an example on configuring a board without having corresponding entries in this file.